modules/helpers/journal.js

```diff
@@ -9,7 +9,7 @@ export function register_dice_enricher() {
     enricher: async (match, options) => {
         let element = document.createElement("span");
         element.classList.add("dietype");
-        element.classList.add(dicetheme);
+        element.classList.add("starwars");
```

Comment:
Why is the `dicetheme` from settings no longer working? This won't fix the symbols for `"genesys"` it'll just hardcode them to the `"starwars"` symbols, which aren't the same.

Reply:
This is actually how it was in the original implementation. I thought I was being smart by using the selected dice theme...

Reply:
Here's some of the original code, for reference - https://github.com/StarWarsFoundryVTT/StarWarsFFG/blob/main/modules/popout-editor.js#L75-L81

Reply:
Ooooh I see, we only want the variable dicetheme when it's a symbol that can vary. Gotchya.